docs/content/stepbystep/Git-setup.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Git setup'
+metaTitle: 'Installing and using git | Discordeno'
+metaDescription: 'Setting up git for your pc'
+---
+
+# Installing
+
+## Installing the CLI
+
+We will now be installing the CLI. This is the terminal interface for git.
+
+First, go to the [Git download page](https://git-scm.com/downloads) and install the version for your OS. I'm using windows so I'll use <https://git-scm.com/download/win>
+
+Once downloaded click next and next again(unless you need to change the install location)
```

Comment:
No space before the bracket :-1:

Reply:
i dont really like spaces before brackets

Reply:
@Suyashtnt Here's the rule: There should be a space before the open parenthesis and no space after it, and there should be a space after the closed parenthesis and no space before it...

Reply:
like this: `![installing gif] (https://i.imgur.com/xvDJ2iI.gif)`

Reply:
@Suyashtnt `![Installing VSC](image-url)`

Reply:
@Suyashtnt I was talking about this:
> Once downloaded click next and next again(unless you need to change the install location)